### ISSUE TITLE
Voice Over in Safari reads required "star"

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#4628]
Voice Over reads, for example , "star first name".

With Chrome, this doesn't happen. This might happen with other screen readers as well.

[+1 from customer](https://mail.google.com/mail/u/0/#inbox/FMfcgxwJXCFxTPzFLLxfkflKNlpxCSvX)

**Comment by ebruchez** [comment]
The question is how to either:

- show the star with CSS `content:` but not let it be read by the screen reader
- show the star with a different mechanism

Using Font Awesome, which uses the "Private Use Area", for the asterisk, VoiceOver on Safari, in particular, doesn't read those.

<img width="382" alt="Screen Shot 2020-08-03 at 10 35 22 AM" src="https://user-images.githubusercontent.com/105769/89212593-91f89280-d578-11ea-8de7-47d63cad3ef0.png">

**Comment by ebruchez** [comment]
This works fine.

Another way to go would be to switch to a system where we don't use asterisks, but [indicate optional fields instead](https://uxdesign.cc/form-field-required-vs-optional-9b4d7cdbf400).

**Comment by ebruchez** [comment]
Some users might still prefer the asterisk, so even if we go the way of #2580, we probably still must keep support for asterisks.

**Comment by ebruchez** [comment]
- [x] basic CSS
- [ ] if the help icon is present, it appears higher up when there is an asterisk (FB app/form name for example)
- [ ] would be good to check with NVDA and Jaws
